.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "python.pythonPath": "/home/juan/.virtualenvs/osubtitles/bin/python",
```

Comment:
Careful with this.

I would ignore this file or remove the `python:pythonPath` key since it seems that [VSCode will use the chosen Python interpreter](https://code.visualstudio.com/docs/python/environments).